tests/Http/Place/UpdateAddressRequestHandlerTest.php

```diff
@@ -0,0 +1,255 @@
+                        new CountryCode('BE')
+                    ),
+                    new Language('nl')
+                ),
+            ],
+            $this->commandBus->getRecordedCommands()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_handles_changing_the_address(): void
+    {
+        $updateAddressRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Veldstraat 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gent",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $updateAddressRequest2 = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Meir 12",
+                    "postalCode": "2000",
+                    "addressLocality": "Antwerpen",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $this->updateAddressRequestHandler->handle($updateAddressRequest);
+        $this->updateAddressRequestHandler->handle($updateAddressRequest2);
+
+        $this->assertEquals(
+            [
+                new UpdateAddress(
+                    self::PLACE_ID,
+                    new Address(
+                        new Street('Veldstraat 11'),
+                        new PostalCode('9000'),
+                        new Locality('Gent'),
+                        new CountryCode('BE')
+                    ),
+                    new Language('nl')
+                ),
+                new UpdateAddress(
+                    self::PLACE_ID,
+                    new Address(
+                        new Street('Meir 12'),
+                        new PostalCode('2000'),
+                        new Locality('Antwerpen'),
+                        new CountryCode('BE')
+                    ),
+                    new Language('nl')
+                ),
+            ],
+            $this->commandBus->getRecordedCommands()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_supports_multiple_languages(): void
+    {
+        $updateAddressRequest = $this->psr7RequestBuilder
+            ->withRouteParameter('placeId', self::PLACE_ID)
+            ->withRouteParameter('language', 'nl')
+            ->withBodyFromString(
+                '{
+                    "streetAddress": "Veldstraat 11",
+                    "postalCode": "9000",
+                    "addressLocality": "Gent",
+                    "addressCountry": "BE"
+                }'
+            )
+            ->build('PUT');
+
+        $updateAddressRequest2 = $this->psr7RequestBuilder
```

Comment:
Nit: I personally don't like adding number suffixes to variable names because that requires understanding the full scope. Maybe adding a language suffix also works, for example: `updateAddressInDutchRequest` and `updateAddressInFrenchRequest` are more